docs/building-fcos.md

```diff
@@ -82,6 +82,19 @@ here need to change for that.
 ```sh
 cosa() {
    env | grep COREOS_ASSEMBLER
+   if [[ -z ${COREOS_ASSEMBLER_CONTAINER} ]]; then
+       cosa_container="quay.io/coreos-assembler/coreos-assembler:latest"
+       cosa_build_date_str="$(podman inspect -f "{{.Created}}" ${cosa_container} | awk '{print $1}')"
+       cosa_build_date="$(date -d ${cosa_build_date_str} +%s)"
+       if [[ "$(date --utc +%s)" -ge $((${cosa_build_date} + 604800)) ]] ; then
+         >&2 echo "${cosa_container} is more that a week old and likely outdated."
+         >&2 echo "You should pull the latest version with:"
+         >&2 echo "podman pull ${cosa_container}"
```

Comment:
```suggestion
         echo "${cosa_container} is more that a week old and likely outdated." >&2 
         echo "You should pull the latest version with:" >&2 
         echo "podman pull ${cosa_container}" >&2 
```

Maybe let's move the stderr redirect to the end of the lines? Just feels awkward to have it in front of the command.